components/ucd/category/src/lib.rs

```diff
@@ -0,0 +1,313 @@
+// Copyright 2012-2015 The Rust Project Developers.
```

Comment:
This line is only needed ifyou are copying content from the Rust project, in which case should have the same years as the original file. Is this the case here?